minorminer/utils/embeddings.py

```diff
@@ -529,17 +576,22 @@ def raster_embedding_search(
 
     for i, f in enumerate(sublattice_iter):
         Tr = _T.subgraph([f(n) for n in tile])
-
-        sub_embs = find_multiple_embeddings(
-            S,
-            Tr,
-            max_num_emb=max_num_emb,
-            timeout=timeout,
-            skip_filter=skip_filter,
-            prng=prng,
-            embedder=embedder,
-            embedder_kwargs=embedder_kwargs,
-        )
+        if tiling:
+            if Tr.number_of_edges() == S.number_of_edges():
```

Comment:
add comment to explain why this special case exists.
probably good idea to briefly describe other cases too.

Reply:
Comment in the docstring isn't good enough?

Reply:
Ooops, missed it in the docstring.